docs/sphinx/applications/python/hybrid_qnns.ipynb

```diff
@@ -28,18 +28,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [],
    "source": [
     "# Install the relevant packages.\n",
     "\n",
-    "!pip install matplotlib==3.8.4 torch==2.2.2 torchvision==0.17.0 scikit-learn==1.4.2 -q"
+    "!pip install matplotlib==3.8.4 torch==2.2.0 torchvision==0.17.0 scikit-learn==1.4.2 -q"
```

Comment:
I'm curious how you knew that lowering the torch version would fix the publishing errors. The publishing errors that I see are something like "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate", so is there some sort of documented SSL incompatibility?